src/components/Search/SearchResultBase.js

```diff
@@ -335,13 +333,13 @@ class SearchResultBase extends Component {
   getFilterById = index => {
     const { getTagById, getSourceById } = this.props;
     switch (index) {
-    case SEARCH_INTENT_INDEX_TOPIC:
-      return getTagById;
-    case SEARCH_INTENT_INDEX_SOURCE:
-      return getSourceById;
-    default:
-      console.log('Using default filter:', index);
-      return x => x;
+      case SEARCH_INTENT_INDEX_TOPIC:
```

Comment:
We've been always formatting it the other way: case under switch. I will accept this one, but myself will continue the old way. Sorry.

Reply:
so pls configure the lint as you wish it should be, because i think it will complain to you...